orchagent/dash/dashhaorch.cpp

```diff
@@ -512,3 +662,116 @@ void DashHaOrch::doTask(ConsumerBase &consumer)
         SWSS_LOG_ERROR("Unknown table: %s", consumer.getTableName().c_str());
     }
 }
+
+void DashHaOrch::doTask(NotificationConsumer &consumer)
+{
+    SWSS_LOG_ENTER();;
+
+    std::string op;
+    std::string data;
+    std::vector<swss::FieldValueTuple> values;
+    
+    consumer.pop(op, data, values);
```

Comment:
You can use `pops` api and process multiple notifications at once. 

Also, Make sure doTask(NotificationConsumer &consumer) is called during the empty doTask() Eg: https://github.com/sonic-net/sonic-swss/blob/90fceadf75654501e5fb2d9726344c6e0d0d0a86/orchagent/orchdaemon.cpp#L882  

if not there is a possibility we might miss some notifications